### ISSUE TITLE
[Bug] Zettlr 2.0 beta: zoom function does not work

### ISSUE THREAD

**Issue by hildebrand-institute** [Zettlr/Zettlr#2558]
### Description

The zoom function does not work neither by unsing `ctrl+/-` nor by using the menu  `View-->Zoom In/Zoom Out`

### Reproducing

See Description

### Zettlr Version

Nightly

### Specify version

2.0.0-beta.5-x86_64+nightly

### Your Platform

- [ ] Windows
- [ ] macOS
- [X] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Fedora 34

**Comment by nathanlesage** [comment]
Does it work if you also press `Shift`? There's a long standing bug in Electron where to Zoom in on non-macOS platforms you have to additional press Shift. But it's good to know that it's apparently not working via the menu, I'll have a look.

**Comment by hildebrand-institute** [comment]
No, there's no way to make it work, not even pressing `Ctrl+Shift...`
Tschüss

**Comment by nathanlesage** [comment]
Hm curious. Can others on Linux/Windows reproduce this?

**Comment by lianghongzhuo** [comment]
On Ubuntu 18.04 I can't reproduce it, the zoom function works fine in beta5

**Comment by nathanlesage** [comment]
Just tested it out on Windows, works well. It might be the keyboard layout? If you're on U.S. ANSI, you have to actually press `Ctrl+Shift+=` (because the `+` is the second key on the equal sign). Closing as non reproducible. If new evidence surfaces, please comment here, then we can reopen it.
